sml-bin/redeploy-testlib.sml

```diff
@@ -0,0 +1,45 @@
+open OS.Path;
+open OS.FileSys;
```

Comment:
I would prefer not to use `open`. Makes some things easier. E.g. when I want to know what `access` does I first have to find out where it comes from (`OS.Path` or `OS.FileSys`, probably not difficult to guess but still easier if it is explicit). What do you think?

Reply:
Since this was a "script", especially originally, I figured using open should be fine.  I am not against using qualified names though.  I will probably make more convenient names aliasing the structures.

Reply:
I switched everything to qualified names now.